conference/src/test/java/org/spine/samples/lobby/conference/ConferenceServiceShould.java

```diff
@@ -0,0 +1,219 @@
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.spine3.base.EmailAddress;
+import org.spine3.base.Event;
+import org.spine3.protobuf.Messages;
+import org.spine3.samples.lobby.common.ConferenceId;
+import org.spine3.samples.lobby.conference.ConferenceInfo;
+import org.spine3.samples.lobby.conference.ConferenceServiceGrpc;
+import org.spine3.samples.lobby.conference.CreateConferenceResponse;
+import org.spine3.samples.lobby.conference.EditableConferenceInfo;
+import org.spine3.samples.lobby.conference.FindConferenceRequest;
+import org.spine3.samples.lobby.conference.UpdateConferenceResponse;
+import org.spine3.samples.lobby.conference.contracts.Conference;
+import org.spine3.samples.sample.lobby.conference.contracts.ConferenceCreated;
+import org.spine3.samples.sample.lobby.conference.contracts.ConferenceUpdated;
+import org.spine3.server.BoundedContext;
+import org.spine3.server.event.EventStore;
+import org.spine3.server.event.EventStreamQuery;
+
+import javax.annotation.Nullable;
+import java.util.LinkedList;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * @author andrii.loboda
+ */
+@SuppressWarnings("InstanceMethodNamingConvention")
+public class ConferenceServiceShould {
+
+    private final BoundedContext boundedContext = Given.boundedContext;
+    private final ConferenceServiceGrpc.ConferenceService conferenceService = Given.getConferenceService();
+    private static final Void NO_RESULT = null;
+
+    private Given given;
+
+    @Before
+    public void setUp() {
+        given = new Given();
+    }
+
+    @After
+    public void tearDown() {
+        given.dropData();
+    }
+
+
+    @Test
+    public void create_conference_and_generate_ConferenceCreated_event() {
+        conferenceService.createConference(given.conferenceInfo(), TestStreamObserver.<CreateConferenceResponse>newBuilder()
+                                                                                     .build());
+
+        final EventStore eventStore = boundedContext.getEventBus()
+                                                    .getEventStore();
+
+        final Function<Event, Void> checkConferenceCreated = new Function<Event, Void>() {
+            @SuppressWarnings("ReturnOfNull")
+            @Override
+            public Void apply(Event event) {
+                assertEquals(Messages.fromAny(event.getMessage())
+                                     .getClass(), ConferenceCreated.class);
+                return NO_RESULT;
+            }
+        };
+        final TestStreamObserver conferenceCreatedObserver = TestStreamObserver.<Event>newBuilder()
+                                                                               .setNextFunction(checkConferenceCreated)
+                                                                               .build();
+        //noinspection unchecked
```

Comment:
Please suppress `unchecked` warnings via annotation because gradle doesn't see comment supperession, it is only for IDEA, and we have warnings in build logs.